connectors/src/connectors/notion/temporal/workflows/upserts.ts

```diff
@@ -127,47 +53,21 @@ export async function performUpserts({
   topLevelWorkflowId: string;
   forceResync: boolean;
 }): Promise<void> {
-  let pagesToSync: string[] = [];
-  let databasesToSync: string[] = [];
-
   const promises: Promise<void>[] = [];
 
-  if (isGarbageCollectionRun) {
```

Comment:
Moved to GC workflow directly, so we could avoid having any GC-specific logic in this file